system/ee/eecli.php

```diff
@@ -62,9 +62,15 @@
 // The $debug value as a constant for global access
 defined('DEBUG') || define('DEBUG', $debug);
 
+// The update command requires some special setup to work in EE 2.x
+if ($argv[1] == 'update') {
+    defined('INSTALL_MODE') || define('INSTALL_MODE', true);
+    define('CLI_UPDATE', true);
+}
+
```

Comment:
This part does not seem to be needed with the changes that I have the CLI map branch, so wondering if we can skip/remove it